agent/php_pdo.h

```diff
@@ -14,6 +13,12 @@
 #include "nr_datastore.h"
 #include "nr_datastore_instance.h"
 
+#if ZEND_MODULE_API_NO >= ZEND_8_5_X_API_NO
+#define PHP_PDO_DBH zend_object*
```

Comment:
Suggest naming this NR_PHP_PDO_DBH to clarify that it's our define vs a PHP define.